docker/userspacecni/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ligato/vpp-base:23.02 as builder
+COPY . /root/userspace-cni-network-plugin
+WORKDIR /root/userspace-cni-network-plugin
+RUN apt update -y 
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y binutils wget make git
+RUN wget -qO- https://golang.org/dl/go1.20.1.linux-amd64.tar.gz | tar -C /usr/local -xz
+ENV PATH="${PATH}:/usr/local/go/bin"
+RUN go mod download
+RUN make generate
+RUN go mod tidy
+RUN make build
+#End of builder container
+
+# Copy build userspace cni bin to a small deployer container
+FROM ubuntu:22.04
```

Comment:
This base image would be quite big?

Reply:
I can change to a smaller image later when I get into a state where I can test all of this running.